query05.sql

```diff
@@ -0,0 +1,27 @@
+-- data: stops.txt - wheelchair_boarding - 0(ukn/inherit),1(yes),2(no)
+
+create index phl_neighborhood_geo_index 
+on neighborhood 
+using gist (the_geom);
+
+
+with bus as (
+    select wheelchair_boarding, 
+        the_geom, stop_id
+    from septa_bus_stops
+),
+
+neighborhood_bus_grouped as (
+    select name,
+        sum(case wheelchair_boarding when 2 then 1 else 0 end) wb_num,
```

Comment:
I think the docs say that `wheelchair_boarding=1` means wheelchair accessible (as opposed to `2`).

Reply:
Yes! You are right! I made a mistake here.